tests/unit/worker/test_submission_worker.py

```diff
@@ -43,7 +123,27 @@ def test_create_dir_as_python_package(self):
 
     def test_return_file_url_per_environment(self):
         returned_url = return_file_url_per_environment(self.url)
-        self.assertEqual(returned_url, "http://testserver/test/url")
+        self.assertEqual(returned_url, "{0}{1}".format(self.testserver, self.url))
+
+    @mock.patch("scripts.workers.submission_worker.SUBMISSION_DATA_DIR", "/tmp/test-dir/compute/submission_files/submission_{submission_id}")
+    @mock.patch("scripts.workers.submission_worker.SUBMISSION_INPUT_FILE_PATH", "/tmp/test-dir/compute/submission_files/submission_{submission_id}/{input_file}")
+    @mock.patch("scripts.workers.submission_worker.create_dir_as_python_package")
+    @mock.patch("scripts.workers.submission_worker.download_and_extract_file")
+    def test_extract_submission_data_successful(self, mock_download_and_extract_file, mock_create_dir_as_python_package):
```

Comment:
can you rename the method to `test_extract_submission_data` or `test_extract_submission_data_success`